### PR TITLE
Update mixed content article to reflect plans to upgrade/block passive mixed content

### PR DESCRIPTION
**Pull request by konklone**
First pass at an update to the mixed content article, in light of increasing HTTPS usage and Chrome's announced plans to autoupgrade or block passive mixed content.

The linked announcement has the original plan, and is being updated as the canonical reference source with any delays or changes to the timetable.

- [ ] This has been reviewed and approved by (NAME)
- [ ] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md [hunk 1]
```diff
@@ -14,18 +14,19 @@ description: Mixed occurs when initial HTML is loaded over a secure HTTPS connec
 connection, but other resources (such as images, videos, stylesheets, scripts)
 are loaded over an insecure HTTP connection. This is called mixed content
 because both HTTP and HTTPS content are being loaded to display the same page,
-and the initial request was secure over HTTPS. Modern browsers display warnings
-about this type of content to indicate to the user that this page contains
-insecure resources.
+and the initial request was secure over HTTPS. 
+
+Modern browsers may block this content, or will display warnings about this type of content to indicate to the user that this page contains insecure resources. Browsers that block mixed content may first attempt to "upgrade" the connection to this content from HTTP to HTTPS.
```
Comment (konklone) [inline src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md:19]: The last sentence refers to "Browsers" but really is just in reference to Chrome's plan. Not sure how these articles like to refer to single-browser initiatives.

File: src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md [hunk 2]
```diff
@@ -14,18 +14,19 @@ description: Mixed occurs when initial HTML is loaded over a secure HTTPS connec
...
 
 ### TL;DR {: .hide-from-toc }
 
 * HTTPS is important to protect both your site and your users from attack.
 * Mixed content degrades the security and user experience of your HTTPS site.
+* Browsers may block mixed content altogether over time, in order to protect your site's users.
```
Comment (konklone) [inline src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md:25]: It seemed significant enough to add it to the top TL;DR bullets, even though so far it's just one browser that's announced this change. Again, not sure how we want to phrase single-browser initiatives.

File: src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md [hunk 3]
```diff
@@ -89,9 +90,11 @@ resource.
...
 Although many browsers report mixed content warnings to the user, by the time
 this happens, it is too late: the insecure requests have already been performed
-and the security of the page is compromised. This scenario is, unfortunately,
-quite common on the web, which is why browsers can't just block all mixed
-requests without restricting the functionality of many sites.
+and the security of the page is compromised. 
+
+This scenario has traditionally been
+quite common on the web, which is why many browsers haven't just blocked all mixed
+requests, so as not to restrict the functionality of many sites. However, as HTTPS has become [overwhelmingly adopted on the web](https://transparencyreport.google.com/https/overview?hl=en), some browsers have announced that they will begin blocking mixed content ([more below](#blocking-optionally-blockable-content)).
```
Comment (konklone) [inline src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md:97]: This is my first PR, and I don't have the site staged locally, so I'm guessing at how anchor tags are autogenerated based on header text, based on experience with past Jekyll-based sites.

**Comment by petele** [comment]
@kaycebasques - FYI since you're potentially migrating this content to web.dev

File: src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md (hunk 1 above)
Comment (estark37) [reply src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md:19]: I'm also not sure what the right precedent is, but fwiw this sounds good to me. We could maybe link to https://www.chromestatus.com/feature/5557268741357568 and https://www.chromestatus.com/feature/4926989725073408 so people can get more up-to-date information about the specifics of what browsers upgrade what content.

File: src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md (hunk 2 above)
Comment (estark37) [reply src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md:25]: What about something like: "Different browsers block different types of mixed content, and Chrome will soon block all mixed content, so it's best to avoid mixed content altogether." (with a link to the blog post)

File: src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md (hunk 1 above)
Comment (rowan-m) [reply src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md:19]: This is under /web, so I'm not too concerned about it being primarily Chrome focused.
